talent/application-process.md

```diff
@@ -0,0 +1,68 @@
+
+- There are notable exceptions to this, such as higher volume junior roles where we directly ask the developer to do the coding challenge before a call.
+
+### Coding Challenge
+
+- Every developer who passes the first interview should complete a coding challenge remotely.
+
+- The coding challenge can be solved in Go/Python/JavaScript/Scala/Ruby.
+
+- Unit tests are provided to validate the correctness of the solution.
+
+- The time limit for solving the coding challenge is 5 hours. 
+
+- We are expecting the solution either in a private Gist or in an attached file.
+
+- The main point being reviewed is the correctness of the solution, however we do check the complexity and code style. 
+
+- In case of the Data Science team, there is an additional ML task necessary to be solved in the time limit of 5 hours. 
+
+- The results are reviewed by the engineering team and feedback is given to the developer as soon as possible. 
+
+
+### On-site Interview
+
+- Every developer who successfully solves the coding challenge, is invited for the last interview at our offices in Madrid. It might last a full day or be split across two half-days, depending on the availability of the developer.
+
+- This is the last step in which it's decided if the developer will receive an offer or not. 
+
+The interviews are the following:
+
+- Kickoff with CTO, QA: to know the developer better.
+
+- Design interview: to see how the developer would design a solution to a given technical problem working with a several of our engineers.
+
+- Logical thinking: to see how the developer solves an abstract problem in a logical way.
+
+- In case of rejection, the developer should be notified in person on the day by the lead of the corresponding team. This should be a constructive discussion giving the developer as much insight as possible. 
+
+### Offer
+
+- The offer is being sent by CTO on Lever. 
+
+- The developer has no time deadline to accept/reject the offer. Only in cases that we have more developers in the offer state for one opening where decisions need to be made. 
+
+### Diversity and Remote
+
+- At source{d} we believe that amazing talent should not be constrained by physical barriers. Our multinational and multicultural team shares a culture of openness and excellence and passion for programming.
+
+- At the moment, we are 20+ people from 10 different countries working closely together from our office in Madrid. We are more than happy to sponsor you a visa and guide you and your family through the whole process if you decide to come to work from our office, but you may also choose to work remotely as part of our team does working from Portugal, Estonia, Russia, and others.
```

Comment:
s/20/25+

s/work remotely as part of our team does working from Portugal, Estonia, Russia, and others./work remotely. Currently, we have remote team members in Portugal, Estonia & Russia.